src/main/java/com/emc/mongoose/Main.java

```diff
@@ -51,44 +37,56 @@
 import java.util.Objects;
 import java.util.stream.Collectors;
 
+import static com.emc.mongoose.Constants.APP_NAME;
+import static com.emc.mongoose.Constants.DIR_EXAMPLE_SCENARIO;
+import static com.emc.mongoose.Constants.DIR_EXT;
+import static com.emc.mongoose.Constants.KEY_CLASS_NAME;
+import static com.emc.mongoose.Constants.KEY_STEP_ID;
+import static com.emc.mongoose.Constants.PATH_DEFAULTS;
+import static com.emc.mongoose.config.CliArgUtil.allCliArgs;
+import static com.emc.mongoose.load.step.Constants.ATTR_CONFIG;
+import static javax.script.ScriptContext.ENGINE_SCOPE;
+import static org.apache.logging.log4j.CloseableThreadContext.Instance;
+import static org.apache.logging.log4j.CloseableThreadContext.put;
+
 public final class Main {
 
-	public static void main(final String... args) {
+	private static Server server;
```

Comment:
use local var for the server instead of the field
field is unnecessary because Main is never instantiated